QEfficient/transformers/cache_utils.py

```diff
@@ -0,0 +1,62 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c)  2023-2024 Qualcomm Innovation Center, Inc. All rights reserved.
```

Comment:
please re-confirm on date ranges on newly added files

Reply:
@quic-jhugo , please confirm the date ranges